utils/auth_utils.py

```diff
@@ -0,0 +1,14 @@
+from decouple import config
+from random import randint
+import requests
+
+def send_otp(contact):
+    otp = str(randint(1000, 9999))
+    url = "http://www.merasandesh.com/api/sendsms"
```

Comment:
add this in settings as well. Because we may need to use another service

Reply:
okay